actions/storage.js

```diff
@@ -62,24 +62,9 @@ export function clear(options = {exclude: []}) {
 }
 
 export function actionOnGlobalItemsWithPrefix(prefix, action) {
-    return (dispatch) => {
-        dispatch({
-            type: StorageTypes.ACTION_ON_GLOBAL_ITEMS_WITH_PREFIX,
-            data: {prefix, action},
-        });
-        return {data: true};
-    };
-}
-
-export function actionOnItemsWithPrefix(prefix, action) {
```

Comment:
This action wasn't actually used anywhere, so I removed it rather than having to also fix it. I'm not really sure what the difference between "items" and "global items" is anyway